features/networking/ovn_ipsec.feature

```diff
@@ -198,70 +198,36 @@ Feature: OVNKubernetes IPsec related networking scenarios
   # @case_id OCP-40569
   @admin
   @destructive
-  @inactive
-  @network-ovnkubernetes @network-openshiftsdn @ipsec
+  @network-ovnkubernetes @network-openshiftsdn
   @4.12 @4.11
   @vsphere-ipi @openstack-ipi @gcp-ipi @baremetal-ipi @azure-ipi @aws-ipi
   @vsphere-upi @openstack-upi @gcp-upi @baremetal-upi @azure-upi @aws-upi
   @proxy @noproxy @disconnected @connected
   @singlenode
   Scenario: OCP-40569:SDN Allow enablement/disablement ipsec at runtime
     Given the env is using "OVNKubernetes" networkType
```

Comment:
Here we check networkType to be OVNKubernetes, but we have tags "@network-ovnkubernetes @network-openshiftsdn". Maybe we should fix this mismatch ?

Reply:
> @network-openshiftsdn

@liangxia Fixed it now.